packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -143,16 +143,16 @@ const config = async (env): Promise<Configuration> => ({
         // If src/README.md exists use it; otherwise the root README
         // To `compiler.options.output`
         { from: hasReadme() ? 'README.md' : '../README.md', to: '.', force: true },
-        { from: 'plugin.json', to: '.' },
+        { from: 'plugin.json', to: '.', context: SOURCE_DIR + '/' },
```

Comment:
CopyWebpackPlugin doesn't support "global context" options so I had no choice but to write multiple context fields.